packages/devextreme/js/__internal/grids/grid_core/sticky_columns/dom.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { CLASSES, StickyPosition } from './const';
+
+const getStickyColumnPosition = (
+  { fixed, fixedPosition }: { fixed: boolean; fixedPosition: string },
+): string => {
```

Comment:
```suggestion
): StickyPosition => {
```
minor
And same in other places